posts/2020-10-12-test-graalvm.md

```diff
@@ -0,0 +1,159 @@
+---
+title: How do we test Micronaut and GraalVM integration?
```

Comment:
I feel like there is a better title for this but I haven't come up with it yet...

Reply:
I'm not happy with it, but not really sure about other alternatives. Maybe @kroegerk can help us :)

Reply:
@ilopmar @JasonTypesCodes 

Suggestions:

Testing Micronaut and GraalVM Integration
A Simple Guide to Testing Micronaut and GraalVM Integration
Testing Micronaut and GraalVM Integration: A Step-by-Step Guide
Testing Micronaut and GraalVM Integration Step-by-Step
Tips for Testing Micronaut and GraalVM Integration
Testing Tips: Micronaut and GraalVM Integration
Testing Micronaut and GraalVM Integration the Easy Way
The Easiest Way to Test Micronaut and GraalVM Integration
Test Your Micronaut and GraalVM Integration with Ease
How to Test Micronaut and GraalVM Integration

Reply:
:exploding_head:

Reply:
Now that I've read over the article more closely, I see that it's not really a tutorial for users, but an explanation of how the Micronaut team tests integration. Some of my suggestions above are not relevant in that case. I'll provide a new list momentarily.

Reply:
New Suggestions:

Testing Micronaut and GraalVM Integration
How Micronaut and GraalVM Integration Is Tested
Micronaut GraalVM Integration: Continual Improvements FTW
Improving Micronaut and GraalVM Integration
Micronaut and GraalVM Integration: The Process and the Results
How We Improved Micronaut GraalVM Integration and What That Means for You
Micronaut GraalVM Integration Yesterday and Today
Improvements to Micronaut GraalVM Integration
Micronaut and GraalVM Integration: Better than Ever
Micronaut GraalVM Integration: A Peek at the Development Process
Micronaut and GraalVM Integration: A Success Story
Micronaut and GraalVM: An Integration Story
Micronaut and GraalVM: An Integration Story and a Happy Ending
Integrating Micronaut and GraalVM: Lessons Learned
Micronaut GraalVM Integration: What We've Learned and How We've Improved the Process
Making Micronaut and GraalVM Play Well Together

Reply:
I like (in no particular order):
- How We Improved Micronaut GraalVM Integration and What That Means for You
- Micronaut GraalVM Integration: A Peek at the Development Process
- Micronaut and GraalVM: An Integration Story and a Happy Ending

@JasonTypesCodes what do you think?

BTW @kroegerk it's amazing how many different options you can think in a few minutes. Next time I need a title for a talk I know who to ask... :rofl: :laughing:

Reply:
I've updated the title.

Reply:
Awww, shucks. Thanks. 

And @ilopmar , I'm always happy to review and make suggestions for anything you write. You know how code just makes sense and "clicks" in your brain? That's how words are in my brain! Please don't hesitate to ask – believe it or not, that kind of thing is fun for me!

Reply:
:heart_eyes: